content/applications/finance/fiscal_localizations/united_states.rst

```diff
@@ -0,0 +1,746 @@
+   - `Authorize.net's ACH payment processing for small businesses documentation
+     <https://www.authorize.net/resources/blog/2021/ach-payments-for-small-businesses.html>`_
+   - :doc:`Setting up Stripe for ACH payments (Odoo) <../payment_providers/stripe>`
+   - `Stripe's ACH Direct Debit documentation <https://docs.stripe.com/payments/ach-debit>`_
+
+.. _l10n_us/nacha:
+
+Send payments: NACHA files
+--------------------------
+
+Odoo can generate a National Automated Clearing House Association (NACHA) compatible |ACH| file to
+send to a company's bank. For each individual *Bank* journal that the company wishes to pay vendors
+with, a |NACHA| configuration section needs to be filled out on the Odoo database.
+
+Configuration
+~~~~~~~~~~~~~
+
+First, navigate to the :menuselection:`Accounting app --> Configuration --> Journals`. Open the
+bank journal and click into the :guilabel:`Outgoing Payments` tab.
+
+.. image:: united_states/us-l10n-nacha-settings.png
+   :align: center
+   :alt: NACHA (National Automated Clearing House Association) configuration settings on Odoo.
+
+.. note::
+   The following |NACHA| configuration information is normally provided by the company's financial
+   institution once they have been approved to send payments via their account.
+
+Under the section labeled, :guilabel:`NACHA configuration` are the fields required to generate a
+|NACHA| compatible |ACH| file to send to a company's bank. First, enter the *routing number* of the
+financial institution in the field labeled, :guilabel:`Immediate Destination`. This information is
+widely available on the Internet and generally varies by bank location. This number is usually
+provided during the initial account setup.
+
+Next, enter the registered name of the financial institution in the field called,
+:guilabel:`Destination`. This information will be provided by the bank or credit union.
+
+Following the :guilabel:`Destination` field is the :guilabel:`Immediate Origin` field. Enter the
+9-digit company ID (Employer Identification Number (EIN)) into this field. This information is
+provided by the financial institution.
+
+Next, the :guilabel:`Company Identification` number should be entered. This is a 10-digit number
+made up of the 9-digit company ID (Employer Identification Number (EIN)), with an additional number
+at the start of the sequence. This number is often a `1`. Check with the financial institution
+should this first number differ to verify that it is correct. This number is provided for |ACH|
+approved accounts.
+
+The :guilabel:`Originating DFI Identification` number is next, this Depository Financial Institution
+(DFI) field contains an assigned 8-digit number. Consult the financial institution for this number.
+
+.. important::
+   Enter the above numbers exactly as the company's financial institution (bank or credit union) has
+   provided them. Failing to do so, would be detrimental to the successful  |NACHA| configuration.
```

Comment:
Wording
```suggestion
   Enter the numerical values in this section *exactly* as the company's financial institution
   (e.g. bank or credit union) has provided them, otherwise risk failing a successful |NACHA|
   configuration in Odoo.
```